website/content/docs/drivers/docker.mdx

```diff
@@ -163,6 +163,23 @@ The `docker` driver supports the following configuration in the job spec. Only
   }
   ```
 
+   <Warning>
+
+   If you are creating your job specification in JSON, you must use an array of
+   maps, not a single map as in HCL.
```

Comment:
Having a single value we're setting in the example makes this ambiguous. Lemme check to verify what this is supposed to look like if we were setting multiple values.

Reply:
Added suggestions with that.